### PR TITLE
RM-8443: Add ExtractPlainText & integrate instead of StripHtmlTags

### PR DESCRIPTION
**Pull request by iam-mholle**
- [x] Deleted StripHtmlTags tests should be adapted for ExtractPlainText
- [x] Search localization attribute and use it instead of the new "vermaehlt" enum value.
- [x] Check if `<br/>` should be replaced with a space character

### CODE REVIEW

**Review by MichaelKetting — APPROVED** [review]
Review done via commits.

**Comment by MichaelKetting** [comment]
- [ ]  Flatten commits

File: Remotion/Web/Core/Utilities/HtmlUtility.cs [hunk 1]
```diff
@@ -50,20 +50,37 @@ public static void HtmlEncode (string nonHtmlString, HtmlTextWriter writer)
...
-      return text.Type == WebStringType.Encoded
-          ? StripHtmlTags(text.GetValue())
-          : text.GetValue();
+      throw new NotSupportedException("Use HtmlUtility.ExtractPlainText instead. (Version 3.0.0)");
+    }
+
+    /// <summary>
+    /// Creates an approximation of <c>innerText</c> for an HTML element. Use this method when creating diagnostic metadata from a <see cref="WebString"/>.
+    /// </summary>
+    public static PlainTextString ExtractPlainText (WebString webString)
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Utilities/HtmlUtility.cs:71]: - [ ] Flatten commits

File: Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocEnumValueControlObjectTest.cs [hunk 2]
```diff
@@ -575,6 +578,19 @@ public void TestSelectOptionPostBack ()
...
     }
 
+    [Test]
+    public void TestSelectOptionWithUmlaut ()
+    {
+      var home = Start();
+
+      var dropDownListBocEnumValue = home.EnumValues().GetByLocalID("MarriageStatusField_DropDownListNormal");
+
+      // select other option first to trigger an for auto-postback on value-changed
```
Comment (a-ctor) [inline Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocEnumValueControlObjectTest.cs:588]:
```suggestion
      // select other option first to trigger an auto-postback on value-changed
```

**Review by a-ctor — APPROVED** [review]

**Pull request merged**